Gemfile.lock

```diff
@@ -69,41 +69,41 @@ GEM
     blacklight_advanced_search (6.3.1)
       blacklight (~> 6.0, >= 6.0.1)
       parslet
-    blacklight_range_limit (6.1.2)
-      blacklight (~> 6.0)
+    blacklight_range_limit (6.3.2)
+      blacklight (>= 6.0.2)
       jquery-rails
       rails (>= 3.0)
     bootstrap-sass (3.3.7)
       autoprefixer-rails (>= 5.2.1)
       sass (>= 3.3.4)
-    bourbon (4.3.4)
+    bourbon (5.0.0)
```

Comment:
Updating bourbon changes the home page layout. Maybe we should pin it to 4.3.4 for now? I'm going to take a look at bouron as part of https://github.com/pulibrary/pulmap/issues/492/